src/main/java/com/almanza/inventory/service/ShoeService.java

```diff
@@ -0,0 +1,18 @@
+package com.almanza.inventory.service;
+
+import com.almanza.inventory.models.ShoeModel;
+import com.almanza.inventory.models.ShoeToAddModel;
+import com.almanza.inventory.models.ShoeToUpdateModel;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.List;
+
+public interface ShoeService {
+
+    ResponseEntity<List<ShoeModel>> getShoes();
```

Comment:
you should return List or collection, never ResponseEntity, that's part of rest layer